.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```
@@ -0,0 +1,103 @@
+name: Bug report
+description: Create a report to help us improve
+#labels: ["bug", "triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **THIS IS NOT A SUPPORT CHANNEL!**
+        **IF YOU HAVE SUPPORT QUESTIONS ABOUT RUNNING OR CONFIGURING YOUR OWN HOME SERVER**,
+        please ask in **[#synapse:matrix.org](https://matrix.to/#/#synapse:matrix.org)** (using a matrix.org account if necessary).
+
+        If you want to report a security issue, please see https://matrix.org/security-disclosure-policy/
+
+        This is a bug report template. By following the instructions below and
+        completing the sections with your information, you will help the us to get all
+        the necessary data to fix your issue.
+
+        You can also preview your report before submitting it. You may remove sections
+        that aren't relevant to your particular case.
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: Describe the problem that you are experiencing
+    validations:
+      required: true
+  - type: textarea
+    id: reproduction_steps
+    attributes:
+      label: Steps to reproduce
+      description: |
+        Describe the series of steps that leads you to the problem.
+
+        Describe how what happens differs from what you expected.
+      placeholder: Tell us what you see!
+      value: |
+        - list the steps
+        - that reproduce the bug
+        - using hyphens as bullet points
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: |
+        IMPORTANT: please answer the following questions, to help us narrow down the problem.
+  - type: input
+    id: homeserver
+    attributes:
+      label: Homeserver
+      description: Was this issue identified on matrix.org or another homeserver?
```

```suggestion
      description: Which homeserver was this issue identified on? (matrix.org, another homeserver, etc)
```
I feel like otherwise this question is a bit confusing because it expects a yes/no and is expected to be answered in a text input (rather than a dropdown/radio).